treeherder/model/data_cycling/signature_remover.py

```diff
@@ -124,9 +124,9 @@ def __extract_properties(signatures) -> List[dict]:
         properties = []
         for signature in signatures:
             signature_properties = {
-                "repository": signature.repository,
-                "framework": signature.framework,
-                "platform": signature.platform,
+                "repository": signature.repository.name,
```

Comment:
Could we add a test for the email' s content? So we don't have to send a real email to check this works as expected.